tests/commands/volgroup.py

```diff
@@ -108,17 +117,18 @@ def runTest(self):
             def_pesize_str = ""
 
         # Pass - correct usage.
-        self.assert_parse("volgroup vg.01 pv.01 --reserved-space=1000",
-                          "volgroup vg.01%s --reserved-space=1000 pv.01\n" % def_pesize_str)
-        self.assert_parse("volgroup vg.01 pv.01 --reserved-percent=50",
-                          "volgroup vg.01%s --reserved-percent=50 pv.01\n" % def_pesize_str)
+        self.assert_parse("volgroup vg.01 pv.01 --reserved-space=1",
+                          "volgroup vg.01%s --reserved-space=1 pv.01\n" % def_pesize_str)
+        self.assert_parse("volgroup vg.01 pv.01 --reserved-percent=1",
+                          "volgroup vg.01%s --reserved-percent=1 pv.01\n" % def_pesize_str)
```

Comment:
This is a boundary value and we really should be testing with boundary values as well. The fact that we weren't was revealed during my mutation testing experiments. In this case it didn't matter which value we were testing with (1 or 50) so I decided to update the value instead of adding a whole new test for it.